common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1754,32 +1765,66 @@ object CelebornConf extends Logging {
     conf.getTimeAsMs("rss.partition.sort.timeout", "220s")
   }
 
-  def partitionSortMaxMemoryRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.partition.sort.memory.max.ratio", 0.1)
-  }
+  val PARTITION_SORTER_DIRECT_MEMORY_RATIO_THRESHOLD: ConfigEntry[Double] =
+    buildConf("celeborn.worker.partitionSorter.directMemoryRatioThreshold")
+      .withAlternative("rss.partition.sort.memory.max.ratio")
+      .categories("worker")
+      .doc("Max ratio of sort memory.")
+      .version("0.2.0")
+      .doubleConf
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0].")
+      .createWithDefault(0.1)
 
-  def workerPausePushDataRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.pause.pushdata.memory.ratio", 0.85)
-  }
+  val WORKER_DIRECT_MEMORY_RATIO_PAUSE_RECEIVE: ConfigEntry[Double] =
+    buildConf("celeborn.worker.directMemoryRatioToPauseReceive")
+      .withAlternative("rss.pause.pushdata.memory.ratio")
+      .categories("worker")
+      .doc("If direct memory usage reaches this limit, worker will stop to receive from executor.")
+      .version("0.2.0")
+      .doubleConf
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0].")
+      .createWithDefault(0.85)
 
-  def workerPauseRepcaliteRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.pause.replicate.memory.ratio", 0.95)
-  }
+  val WORKER_DIRECT_MEMORY_RATIO_PAUSE_REPLICATE: ConfigEntry[Double] =
+    buildConf("celeborn.worker.directMemoryRatioToPauseReplicate")
+      .withAlternative("rss.pause.replicate.memory.ratio")
+      .categories("worker")
+      .doc("If direct memory usage reaches this limit, worker will stop to receive from executor and other worker.")
```

Comment:
`If direct memory usage reaches this limit, the worker will stop to receive replication data from executor and other workers.`

Reply:
> ... to receive replication data ...

Should we emphasize "replication" here?